RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -1817,7 +1817,7 @@ boolean L11_hiddenCity()
 		boolean workingHoliday = (turnsUntilWorkingHoliday == 0);
```

Comment:
```suggestion
		boolean workingHoliday = (turnsUntilWorkingHoliday == 0 || auto_haveQueuedForcedNonCombat()) ;
```

since now it can come back to these tasks with deployed spikes
boolean elevatorAction too should be set true in line 1680 if noncombat forced?

In hidden city if the expected method to force noncombat is to start a fight it should affect the decisionmaking to force, or drink Cursed Punch, but that seems complicated to implement

Reply:
Another good call. Updated both workingHoliday and elevatorAction